vaultenv.nix

```diff
@@ -53,20 +53,20 @@ mkDerivation {
     hpack .
   '';
 
-  postInstall = ''
-    # By default, a Haskell derivation contains extra stuff that causes the output to retain
-    # a dependency on GHC and other build tools. Those are not necessary for running the
-    # executable, so we get rid of that.
-    rm -rf $out/lib
-  '';
+  # postInstall = ''
+  #   # By default, a Haskell derivation contains extra stuff that causes the output to retain
+  #   # a dependency on GHC and other build tools. Those are not necessary for running the
+  #   # executable, so we get rid of that.
+  #   rm -rf $out/lib
+  # '';
```

Comment:
Was this intended to be commented out? It sounds important to keep the shipped derivation small.

Reply:
I have no clue why @FPtje commented this out, it seems to work fine when not commented out.